sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -23,6 +23,12 @@ export class Application extends RecordDataModel {
   })
   data: any;
 
+  @Column({
+    name: "application_number",
+    nullable: false,
```

Comment:
So far, we believe that the application number will be generated only during application submission what means that while saving a draft application the number will not be generated and the column must accept null.
I would recommend removing the "nullable: false" to not lead a developer to false assumptions.

Reply:
okay, So the draft application won't have the application number?